vowpalwabbit/offset_tree_internal.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <cstdint>
+#include <vector>
+#include "learner.h"
+namespace offset_tree
+{
+  struct tree_node {
+    tree_node(uint32_t i, uint32_t l, uint32_t r, bool lf);
```

Comment:
Can you use more descriptive names here?

Reply:
done